tests/manage/test_create_storageclass_with_same_name.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import pytest
+
+from ocs import defaults
+from ocsci.testlib import tier1, ManageTest
+from resources.ocs import OCS
+from ocs.exceptions import CommandFailed
+from ocsci.config import ENV_DATA
+
+log = logging.getLogger(__name__)
+
+SC_OBJ = None
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    This fixture defines the teardown function.
+    """
+
+    def finalizer():
+        teardown()
+
+    request.addfinalizer(finalizer)
+
+
+def teardown():
+    """
+    Tearing down the environment
+
+    """
+    log.info(f"Deleting created storage class: {SC_OBJ.name}")
+    SC_OBJ.delete()
+    log.info(f"Storage class: {SC_OBJ.name} deleted successfully")
+
+
+def create_storageclass(sc_name, expect_fail=False):
+    """
+    Function to create a storage class and check for
+    duplicate storage class name
+
+    Args:
+       sc_name (str): name of the storageclass
+        expect_fail (bool): To catch the incorrect scenario in OC - in case
```

Comment:
Wrong indention.

Reply:
thanks.

Reply:
yes

Reply:
Its generally due to mix of TAB and SPACE

Reply:
@fbalak done. Thanks :)